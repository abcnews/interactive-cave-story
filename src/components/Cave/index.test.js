const React = require('react');
const renderer = require('react-test-renderer');

const Cave = require('.');

describe('Cave', () => {
  test('It renders', () => {
    const component = renderer.create(<Cave config={{ shift: 0, x: 0, y: 0 }} />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
