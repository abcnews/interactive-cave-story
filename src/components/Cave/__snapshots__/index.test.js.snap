// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cave It renders 1`] = `
<div
  className="wrapper"
>
  <div
    className="transformer"
    style={
      Object {
        "height": 1656.8259587020648,
        "transform": "translateY(50px)",
        "width": 1024,
      }
    }
  >
    <img
      src="cave.svg"
      style={
        Object {
          "height": 1656.8259587020648,
          "maxWidth": "1000%",
          "width": 1024,
        }
      }
    />
  </div>
</div>
`;
